content/en/blog/2023/otel-in-focus-09.md

```diff
@@ -0,0 +1,213 @@
+alternate `io.opentelemetry.semconv:opentelemetry-semconv:1.21.0-alpha` is now
+introduced from a new repository.
+
+[Java Instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.30.0)
+brings significant changes including the addition of new JavaAgent
+instrumentation, enhancements, and bug fixes. Important changes include the
+splitting of experimental HTTP server metrics into a separate class, renaming of
+`HttpClientResend` and `HttpRouteHolder` to `HttpClientResendCount` and
+`HttpServerRoute`, and removal of a deprecated configuration.
+
+New Java Agent instrumentation has been added for hibernate reactive.
+Enhancements encompass support for AWS Secrets Manager JDBC URLs, improved
+support for semantic convention changes, addition of `javaagent` to
+instrumentation BOM, and more. Several bugs have also been fixed, such as issues
+with `getDefinedPackage` lookup for OpenJ9, serializing key with Lettuce
+instrumentation, and auto-instrumentation with JMX not working without a
+trigger.
+
+#### [Go](/docs/instrumentation/go/)
+
+[Version 1.19.0](https://github.com/open-telemetry/opentelemetry-go/releases/tag/v1.19.0)
+is the first stable release of Metrics and the standard project stability
+guarantees now apply to the `go.opentelemetry.io/otel/sdk/metric` package.
+
+Added features include a new "Roll the Dice" application example and the
+`WithWriter` and `WithPrettyPrint` options to customize `io.Writer` and display
+output in human-readable JSON format.
+
+Notable changes comprise allowance of '/' characters in metric instrument names
+and the change in the default output format of the exporter, which is now more
+compact.
+
+Fixed issues include a recurring problem where the SDK attempted to create the
+Prometheus metric at each Collect, even if known that the scope was invalid. In
+terms of removals, the
+`go.opentelemetry.io/otel/bridge/opencensus.NewMetricExporter` has been replaced
+by `NewMetricProducer`.
+
+Due to the plethora of content, detailed information is advised to be obtained
+by referring to the
+[Full Changelog](https://github.com/open-telemetry/opentelemetry-go/compare/v1.18.0...v1.19.0).
+
+Please note that v1.18.0 included several deprecations and removals:
+
+- Jaeger exporters and examples have been removed, as Jaeger accepts native OTLP
+  now.
+- Go compatibility before version 1.20 is no longer guaranteed.
+
+#### [JavaScript](/docs/instrumentation/js/)
+
+[Version 1.17.0](https://github.com/open-telemetry/opentelemetry-js/releases/tag/v1.17.0)
+fixes an unintentional breaking change in previous releases.
+
+There have been several important changes in the experimental packages,
+including deprecation of legacy configuration APIs.
+
+#### [.NET](/docs/instrumentation/net/)
+
+[Version 1.6.0](https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/core-1.6.0)
+includes minor updates to metrics.
```

Comment:
How about adding a section (or subsection for OpenTelemetry .NET Automatic Instrumentation)

In September it reached GA. See https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/ and https://opentelemetry.io/blog/2023/otel-dotnet-auto-instrumentation/

CC @open-telemetry/dotnet-instrumentation-maintainers